web/html/src/components/icontag.js

```diff
@@ -85,13 +85,15 @@ function IconTag(props) {
     "item-download-csv": "fa spacewalk-icon-download-csv",
     "item-edit": "fa fa-edit",
     "item-enabled": "fa fa-check text-success",
+    "item-enabled-pending": "fa fa-hand-o-right text-success",
```

Comment:
Really list picky comment: does the `pending` status generate any inconsistency or some unexpected behavior we should *warn* the user? If so, I'd better use the `text-warning`, only in this case. It will be more highlighted among all other *green* checks. What do you think?

For the `item-error-pending` the `text-danger` makes more sense instead. :+1:

Reply:
`text-success` is to show the icon with green. Since the same icon (fa-hand-o-right) is used for both enabled-but-pending and for disabled-but-pending states I think it would be a bit confusing to use some other color except green (or red for disabled).

Reply:
Ack, I overlooked the fact it is used in both cases, for *enabled* and *disabled*.

I then agree, and I'd summarize it with these two meanings:
1- the green means everything is working fine anyway
2- the hand icon means an action is pending over here, even if everything is still fine as per point 1

Correct?
Then, :+1:

Reply:
That's correct.